src/UIOMatic/App_Plugins/UIOMatic/backoffice/uioMaticTree/list.html

```diff
@@ -19,9 +19,25 @@
         <div class="umb-panel-body umb-scrollable row-fluid">
             <div class="tab-content form-horizontal" style="padding-bottom: 90px">
                 <div class="umb-pane">
+                    <umb-control-group ng-repeat="property in filterproperties" label="{{property.Name}}" description="{{property.Description}}">
+                        <div>
+                            <select ng-model="property.OperatorID"
+                                    data-ng-options="operator.id as operator.name for operator in Operators"></select>
+                            <div ng-include="property.View"></div>
+                        </div>
+                        <!--div ng-include="property.View"></div-->
+                    </umb-control-group>
+
+                    <div class="btn-group" ng-show="isFilterForm">
+                        <a class="btn btn-info" ng-click="filter()" prevent-default="">
+                            <localize key="actions_filter">Query</localize>
```

Comment:
@serenabenny just remove the localise element, and keep the inner text, we won't be translating the UI so no need for it :)